app/pods/components/cases/new-case/component.js

```diff
@@ -23,19 +27,33 @@ export default Component.extend({
         created: newDate,
       });
     await caze.save();
-    this.set('isLoading', false);
-    return this.close(caze);
-  },
+    this.isLoading = false;
+    return this.args.close(caze);
+  }
 
-  actions: {
-    async createCase($event) {
-      this.set('isLoading', true);
-      $event.preventDefault();
+  get getClassForShortTitle() {
+    if (this.hasError) {
+      return 'auk-form-group--error';
+    }
+    return null;
+  }
+
+  @action
+  async createCaseAction($event) {
+    $event.preventDefault();
+    const {
+      shortTitle,
+    } = this;
+    if (shortTitle < 1) {
```

Comment:
is er een soort van manier dat je kan testen dat er op zijn minst 1 woord gebruikt wordt ?
Het is maar een bedenking, nu kan je nog steeds dossiers ingeven zonder deftige titel.
De validatie is goed, nu nog de voorwaarden ervan

Reply:
Fixed